jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -72,19 +72,21 @@ def gitFetch(url, gitTokenId) {
 
 //Gets the latest release by git tags
 def getLatestReleaseByTag(gitOrg, gitRepo, gitTokenId, productName) {
+    def gaReleaseTagRef = params.gaReleaseTagRef
     def latestRelease = ""
     def url = "https://api.github.com/repos/${gitOrg}/${gitRepo}/git/refs/tags"
 
     def (code, releases) = gitFetch(url, gitTokenId)
     if (code != 200) {
-        error "[ERROR] Failed to retrieve tags for:${productName}. The :${gitRepo} repository does not have any tags available"
+      error "[ERROR] Failed to retrieve tags for:${productName}. The :${gitRepo} repository does not have any tags available"
     }
 
-    if (productName == "fuse") {
-        // Retrieve only Fuse's GA releases
-        releases = releases.findAll { release -> release.ref.contains("-redhat-") }
+    if (!gaReleaseTagRef) {
+      gaReleaseTagRef = ""
     }
```

Comment:
You would be better to move this to outside the method and declare the variable with everything else here https://github.com/integr8ly/ci-cd/blob/master/jobs/release-monitoring/discovery/github/Jenkinsfile#L238 . You can also just declare it like this to avoid lines 84-85:

```
​def gaReleaseTagRef = params.gaReleaseTagRef ?: ""
```